### ISSUE TITLE
Error installing countly-server on ubuntu 20.04

### ISSUE THREAD

**Issue by NTMS2017** [Countly/countly-server#1746]
Hi,

Installation stuck on "Installing views plugin". Last night 23:00 to 09:00 waited in same part. I created a clean ubuntu 20.04. I install all the requirements and check 3 times and restart and checked it again if they works.

Any idea why is not working?

There is 2 way to install county-server community edition based on these documentation;

::: https://support.count.ly/hc/en-us/articles/360036862332-Installing-the-Countly-Server
**Method 1: One liner installation**
The following command will download and install the Countly Community Edition on your Ubuntu or CentOS server.
```
sudo su - 
wget -qO- http://c.ly/install | bash
```

**Method 2: Manualy Download from** (https://github.com/Countly/countly-server)
Download the latest Countly Community Edition server code (includes everything required to run the Countly Community Edition on your server, e.g. dashboard and application server).

First of all the Method 1 has 6207 files and downloaded folder has 6211 files. I an not sure if there is a problem in file size.

I use Method 1: and installation stuck on "Installing views plugin". Than I delete the /root/countly/ files. Download files using Method 2: and paste into /root/countly/. Again installation stuck on "Installing views plugin".

Here is my Ubuntu 20.04 prerequest:

Requirements:
```
niyazitoros@ubuntu:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

niyazitoros@ubuntu:~$ nginx -v
nginx version: nginx/1.18.0 (Ubuntu)

niyazitoros@ubuntu:~$ sudo systemctl status nginx
[sudo] password for niyazitoros: 
● nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2021-02-16 09:16:12 EET; 1h 4min ago
       Docs: man:nginx(8)
   Main PID: 2890 (nginx)
      Tasks: 5 (limit: 7031)
     Memory: 6.3M
     CGroup: /system.slice/nginx.service
             ├─2890 nginx: master process /usr/sbin/nginx -g daemon on; master_process on;
             ├─2891 nginx: worker process
             ├─2892 nginx: worker process
             ├─2893 nginx: worker process
             └─2894 nginx: worker process

Feb 16 09:16:12 ubuntu systemd[1]: Starting A high performance web server and a reverse proxy ser>
Feb 16 09:16:12 ubuntu systemd[1]: Started A high performance web server and a reverse proxy serv>

niyazitoros@ubuntu:~$ node -v
v14.15.5

niyazitoros@ubuntu:~$ npm -v
6.14.11

niyazitoros@ubuntu:~$ mongo --version
MongoDB shell version v4.4.3
Build Info: {
    "version": "4.4.3",
    "gitVersion": "913d6b62acfbb344dde1b116f4161360acd8fd13",
    "openSSLVersion": "OpenSSL 1.1.1f  31 Mar 2020",
    "modules": [],
    "allocator": "tcmalloc",
    "environment": {
        "distmod": "ubuntu2004",
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}

niyazitoros@ubuntu:~$ sudo systemctl status mongod
● mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; disabled; vendor preset: enabled)
     Active: active (running) since Tue 2021-02-16 09:44:24 EET; 37min ago
       Docs: https://docs.mongodb.org/manual
   Main PID: 11142 (mongod)
     Memory: 60.1M
     CGroup: /system.slice/mongod.service
             └─11142 /usr/bin/mongod --config /etc/mongod.conf

Feb 16 09:44:24 ubuntu systemd[1]: Started MongoDB Database Server.
Feb 16 09:44:59 ubuntu systemd[1]: /lib/systemd/system/mongod.service:12: PIDFile= references a p>

```

**Comment by NTMS2017** [comment]
- I red MongoDB issue in here (https://github.com/Countly/countly-server/issues/1649) but didn't give me clear solution. 
- I also checked (https://github.com/Countly/countly-server/issues/1684) but I am not sure if solution fixed, because there is no information.

**Comment by ar2rsawseen** [comment]
Hello,
potentially, it seems it cannot connect to MongoDB specifically.

If you simply run command `mongo` does it connect to MongoDB normally?

**Comment by NTMS2017** [comment]
I am not sure why it cannot connect. But my mongodb app and runming. I did use mongo shell. It works; db.help(), db.version() etc. I also install MongoDB Compas to see if connection works. In MongoDB Compas main screem I hit the connect button and it worked. 

After the last night and this morrining before above post I did try 2 time. Now (20 min ago) after testing with compas I use method 1 to install and it seems last 5 min. Stuck on “installing views plugin”.

I am not sure where to look and determine why stucks there?
